public/data/web3community.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "Web3 Community",
+  "bio": "An Open Source Community that focuses on decentralized applications, web 3, and blockchain technologies 🚀",
+  "avatar": "https://github.com/web3community.png",
+  "links": [
+    {
+      "name": "Visit Website",
+      "url": "https://web3community.github.io",
+      "icon": "globe"
+    },
+    {
+      "name": "Read our Blog",
+      "url": "https://dev.to/web3community",
+      "icon": "book"
+    },
+    {
+      "name": "Mail us",
```

Optional, just an idea. 😃
```suggestion
      "name": "Contact us via mail",
```